SU-SWS/ace-sdssgryphon
# SDSS-234: decanter update

# READY FOR REVIEW

# Summary
- Updating decanter templates

# Review By (Date)
- 3/1

# Urgency
- Normal

# Steps to Test

1. pull in this change.
2. Refresh
3. See the `title` attribute is gone from the footer.
4. Note that the non-discrimination link changed as part of a UComm update to Decanter.

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
- [SDSS-234](https://stanfordits.atlassian.net/browse/SDSS-234)
- [UComm-1944](https://stanfordits.atlassian.net/browse/UCP-1944)


[SDSS-234]: https://stanfordits.atlassian.net/browse/SDSS-234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
